src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.macios.cs

```diff
@@ -15,14 +17,15 @@ public class MauiMediaElement : UIView
 	/// <param name="playerViewController">The <see cref="AVPlayerViewController"/> that acts as the platform media player.</param>
 	/// <param name="parentViewController">The <see cref="UIViewController"/> that acts as the parent for <paramref name="playerViewController"/>.</param>
 	/// <exception cref="NullReferenceException">Thrown when <paramref name="playerViewController"/><c>.View</c> is <see langword="null"/>.</exception>
+	[Obsolete("Depracated CTOR. UIViewController? parentViewController is obsolete")]
 	public MauiMediaElement(AVPlayerViewController playerViewController, UIViewController? parentViewController)
 	{
 		ArgumentNullException.ThrowIfNull(playerViewController.View);
 		playerViewController.View.Frame = Bounds;
 
 #if IOS16_0_OR_GREATER || MACCATALYST16_1_OR_GREATER
 		// On iOS 16+ and macOS 13+ the AVPlayerViewController has to be added to a parent ViewController, otherwise the transport controls won't be displayed.
-		var viewController = parentViewController ?? WindowStateManager.Default.GetCurrentUIViewController();
+		var viewController = WindowStateManager.Default.GetCurrentUIViewController();
```

Comment:
Can you revert the changes on this ctor? This is still a breaking change.

Reply:
Yes I will do that now.

Reply:
@pictos Updated as requested